### ISSUE TITLE
Enhancement: Update Kallisto wrapper adding option to save stdout as a dataset for MultiQC use

### ISSUE THREAD

**Issue by jennaj** [galaxyproject/tools-iuc#6610]
toolshed.g2.bx.psu.edu/repos/iuc/kallisto_quant/kallisto_quant/0.48.0+galaxy1

MultiQC will parse the **stdout** from Kallisto (only). The Galaxy wrapper for MultiQC seems ready to do this, but the file from Kallisto is not yet available. This request is to add an option to the Kallisto tool form to write that log as a dataset in the history for use in a workflow.

Xref
* Reported at https://help.galaxyproject.org/t/using-multiqc-with-kallisto/14075
* How the tool works https://github.com/MultiQC/MultiQC/issues/440#issuecomment-1024615275
* Prior PRs mentioned in https://github.com/galaxyproject/tools-iuc/pull/1821

**Alternatively**: It would be super interesting to write a completely new tool that captures and writes out the stderr and/or stdout and/or info and/or description content -- from any dataset -- into the history as separate txt files. Maybe include a "stats" output with a checksum, timestamps, server/cluster details, anything else easily captured out of the technical metadata. Multi-use, we don't even need to know or care why. This probably needs a different ticket if anyone else likes the idea.

**Comment by bernt-matthias** [comment]
> It would be super interesting to write a completely new tool that captures and writes out the stderr and/or stdout and/or info and/or description content -- from any dataset

Yes, but I my knowledge Galaxy only stores only a prefix of a certain length in the DB.
